frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -741,7 +747,12 @@ export class NativeQueryEditor extends Component<
       resizable,
       editorContext = "question",
       setDatasetQuery,
-      sidebarFeatures,
+      sidebarFeatures = {
+        dataReference: true,
+        variables: true,
+        snippets: true,
+        promptInput: true,
+      },
```

Comment:
This component already defines default `sidebarFeatures` in `defaultProps`: https://github.com/metabase/metabase/blob/8a26dc03b4fa82c11596a7e63c8f31edfcf6592e/frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx#L217-L228

Reply:
yeah, but it doesn't make `NativeQueryEditorSidebar` happy because TypeScript doesn't look into `defaultProps` :/

Not sure what's the best approach here, maybe making `features` optional and defining a default value in `NativeQueryEditorSidebar` itself? WDYT @kamilmielnik ?